### PR TITLE
Issue template

### PR DESCRIPTION
**Pull request by mts299**
Hi again, 

Here are some issue templates that we can use. Let me know what you think.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# BUG 
+
+*description of the bug*
+
+## Priority
+
+- [ ] Minor - typo, incorrect naming, fails on specific uncommon situation
+- [ ] Major - miss representation of data, failure in more common situation 
+- [ ] Urgent - failure in usage, large miss representation
```
Comment (aburrell) [inline .github/ISSUE_TEMPLATE/bug_template.md:9]:
```suggestion
- [ ] Major - misrepresentation of data, failure in a common situation 
- [ ] Urgent - failure in usage, large misrepresentation
```

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+### Attempts
+
+*What have you tried already to fix it?*
+
+*What have you tried to get around it?*
+
+### Data Location
+
+*Please indicate where to get the data you found the bug with.*
```
Comment (aburrell) [inline .github/ISSUE_TEMPLATE/bug_template.md:23]:
```suggestion
*Please indicate where (and how) to get the data file that was used to obtain the bug.*
```

File: .github/ISSUE_TEMPLATE/deprecation_template.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ 
+**package** */directory*
+
+## Reason
+
+*Why should this feature be deprecated?*
+
+*Is it out of pyDARN's scope?*
+
+*Legacy?*
```
Comment (aburrell) [inline .github/ISSUE_TEMPLATE/deprecation_template.md:15]:
```suggestion
*Why should this feature be deprecated?  Possible reasons include being outside of pyDARN's current scope or being a legacy function that is no longer used.*
```

File: .github/ISSUE_TEMPLATE/feature_template.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+*Please provide a description of the new feature and if will improve or solve a problem.*
+
+### pyDARN Checklist 
+
+-  [ ] Does this fit within pyDARN's scope? 
+-  [ ] Is this a minor change?
+-  [ ] Is this a major change?
+-  [ ] Do you need help developing it? 
+-  [ ] Have you created a pyDARN project to reflect the process of developing it?
```
Comment (aburrell) [inline .github/ISSUE_TEMPLATE/feature_template.md:27]: What does this mean?  I don't understand what this last point is trying to say, and so suggest rewording it.

File: .github/ISSUE_TEMPLATE/feature_template.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+-  [ ] Does this fit within pyDARN's scope? 
+-  [ ] Is this a minor change?
+-  [ ] Is this a major change?
+-  [ ] Do you need help developing it? 
+-  [ ] Have you created a pyDARN project to reflect the process of developing it? 
+
+### Developement help
+
+*pyDARN does not have a dedicated developement team on hand to full feature developement.
```
Comment (aburrell) [inline .github/ISSUE_TEMPLATE/feature_template.md:31]:
```suggestion
*pyDARN does not have a dedicated development team on hand to full feature development. 
```

File: .github/ISSUE_TEMPLATE/feature_template.md (hunk 4 above)
Comment (mts299) [reply .github/ISSUE_TEMPLATE/feature_template.md]: Can you check out the new changes to see if this list is more clearer?

**Comment by mts299** [comment]
@aburrell does the changes look good now? 

Ignore Travis Windows is crashing because an SSL cert not being updated

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+*What have you tried to get around it?*
+
+### Data Location
+
+*Please indicate where (and how) to get the data file that was used to obtain the bug.*
+
+## Potential Bug Location 
+
+*Please note this is not required.*
+*If you can link to some code in pyDARN where you think the bug is coming from this will help the developer fix it sooner*
```
Comment (aburrell) [inline .github/ISSUE_TEMPLATE/bug_template.md:28]:
```suggestion
*If you can link to some code in pyDARN where you think the bug is coming from this will help the developer fix it sooner.*
```

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+## Potential Bug Location 
+
+*Please note this is not required.*
+*If you can link to some code in pyDARN where you think the bug is coming from this will help the developer fix it sooner* 
+
+## Potential Solution(s)
+
+*Please note this is not required.*
+*If you have some idea of how to fix/improve on the bug then please describe here for possible solution(s)*
```
Comment (aburrell) [inline .github/ISSUE_TEMPLATE/bug_template.md:33]:
```suggestion
*If you have some idea of how to fix/improve on the bug then please describe here for possible solution(s).*
```

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+## Extra Notes
+
+*Please provide any other details about this feature*
+- *Operating System*
+- *python version 3.X.X*
+- *examples of what the visual should look like*
+- *other data files that show the same bug*
+- *How to access data files to test out the bug*
```
Comment (aburrell) [inline .github/ISSUE_TEMPLATE/bug_template.md:42]:
```suggestion
*Please provide other pertinent details about this feature:*
- *Operating System*
- *Python version 3.X.X*
- *Examples of what the visual output should look like*
- *Other data files that show the same bug*
- *How to access data files to test out the bug*
```

File: .github/ISSUE_TEMPLATE/deprecation_template.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+
+*Why should this feature be deprecated?  Possible reasons include being outside of pyDARN's current scope or being a legacy function that is no longer used.*
+
+## Replacement
+
+*Would anything replace this feature?*
+
+## Details
+
+*Please provide any other details about the deprecation process and the future.*
```
Comment (aburrell) [inline .github/ISSUE_TEMPLATE/deprecation_template.md:19]:
```suggestion
*Please provide any other details about the deprecation process, such as a timeline for warning the user about the deprecation.*
```

Rewording suggestion to provide a possible clarification.  May or may not be useful.

File: .github/ISSUE_TEMPLATE/feature_template.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
+# Requesting New Feature
+
+**Name**: *Feature name*
+
+**module**:  *potential_filename.py* 
+
+**package**: */which directory should it go in*
```
Comment (aburrell) [inline .github/ISSUE_TEMPLATE/feature_template.md:7]:
```suggestion
**package**: *which directory should it go in*
```

File: .github/ISSUE_TEMPLATE/feature_template.md (hunk 4 above)
Comment (aburrell) [reply .github/ISSUE_TEMPLATE/feature_template.md]: It is clearer now.

**Comment by mts299** [comment]
@aburrell added in all your suggestions

**Comment by aburrell** [comment]
Looks like the CI failure isn't related to the files here, so happy to merge and delete branch.

**Comment by mts299** [comment]
Yea... one error is from numpy update and another is about certifi fixed in other PR's but the travis file doesn't merge over...
